README.md

```diff
@@ -41,6 +41,7 @@ For implementing `keyExtractor`, see `getSectionProps` below.
 | items                | The items of the list, either strings or objects with titles.                                                                                                                                                                                | (string \| { title: string; key?: string \| number; })[] | yes      |
 | indexItemHeight      | The height if an item in the index list (must be a constant). defaults to 25px.                                                                                                                                                              | number                                                   | no       |
 | indexWrapperStyle    | The style of the view that contains the index list.                                                                                                                                                                                          | ViewStyle                                                | no       |
+| indexTextStyle    | The style of the view that contains the index text.                                                                                                                                                                                         | ViewStyle                                                | no       |
```

Comment:
Sorry I just noticed this, but indexTextStyle is a TextStyle, not a ViewStyle. Could you change the type and the description to match this?
Other then this, all looks good 👍

Reply:
sure. Updated.